.github/CODEOWNERS

```diff
@@ -575,7 +575,7 @@ sdk/trafficmanager/Microsoft.Azure.Management.TrafficManager/            @tmsupp
 /sdk/search/                                    @brjohnstmsft @arv100kri @bleroy @tg-msft
 
 # ServiceLabel: %Search %Service Attention
-/sdk/search/Microsoft.*/                        @brjohnstmsft @arv100kri @bleroy @tjacobhi @markheff @miwelsh
```

Comment:
Can you help me understand why these changes are being made?

Reply:
I reached out via email. @brjohnstmsft no longer works on the search team. @markheff and @miwelsh are not linked to any internal aliase